pkg/gclient/pubsub/client.go

```diff
@@ -75,6 +75,11 @@ func (c *pubsubClient) CreateSubscription(ctx context.Context, id string, cfg Su
 	return &pubsubSubscription{sub: sub}, nil
 }
 
+// Topic implements pubsub.Client.Topic
```

Comment:
Yes, this was introduced because the pubsub client couldn't be unit tested. I'll create an issue for investigating switching to pstest.

Note the gclient folder contains mocks for other GCP services besides pubsub.